src/Orleans/Core/Grain.cs

```diff
@@ -269,9 +269,10 @@ protected Grain()
         /// This constructor is particularly useful for unit testing where test code can create a Grain and replace
         /// the IGrainIdentity, IGrainRuntime and State with test doubles (mocks/stubs).
         /// </summary>
-        /// <param name="state"></param>
         /// <param name="identity"></param>
         /// <param name="runtime"></param>
+        /// <param name="state"></param>
+        /// <param name="storage"></param>
```

Comment:
if you are not filling the parameters (or `returns` tag), just remove them altogether. It does get rid of the warning, and causes less maintenance issues in the future (if you add new parameters), as well as avoids overriding explicitly with empty in the case the member is inherited. What does cause a warning though is having a few parameters *documented* but not all